docs/security-considerations.rst

```diff
@@ -239,6 +236,22 @@ Minor Details
 Recommendations
 ***************
 
+Take Warnings Seriously
+=======================
+
+If the compiler warns you about something, you should better change it.
+Even if you do not think that this particular warning has security
+implications, there might be another issue buried beneath it.
+Any compiler warning we issue can be silenced by slight changes to the
+code.
+
+Also try to enable the "0.5.0" safety features as early as possibly
+by adding ``pragma experimental "v0.5.0";``. Note that the word
+``experimental`` does not mean that the safety features are in any
```

Comment:
Should make it clear that holds only true for `0.5.0` and not other experimental pragmas.